src/js/types/BoundedBlsSignatureRequestContextRequest.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 - MATTR Limited
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A context to create a blind BBS signature request
+ */
+export interface BoundedBlsSignatureRequestContextRequest {
+  /**
+   * The resulting commitment of the blinded messages to sign
+   */
+  readonly issuerPublicKey: Uint8Array;
```

Comment:
```suggestion
  readonly signerPublicKey: Uint8Array;
```
It seems more appropriate to use "signer" rather than "issuer" in this layer, as shown in [IETF BBS+](https://datatracker.ietf.org/doc/draft-irtf-cfrg-bbs-signatures/) or [DIF Bound BBS+](https://identity.foundation/bbs-signature/draft-blind-bbs-signatures.html)